source/develop/release-management/features/image-upload.html.md

```diff
@@ -1,232 +1,217 @@
 
-The goal of this project is to simplify the workflow of introducing new ISOs or disk images into the oVirt environment by providing a browser-based UI and REST API through which uploads can be performed. These new methods will support uploading files located on the admin's local machine (e.g. with a file selection dialog in webadmin) or by specifying a remote URL from which oVirt should download the file and place it on the storage domain.
+Before introducing this feature, placing images on oVirt's domains involved using the command line, either via the ISO/Disk Uploader tool or manual copying of an ISO image to its domain. Similarly, importing a VM disk image to a data domain requires an import from existing storage, virt-v2v operation, or other manual method. On the other way, it wasn't possible to expose these images in any way except manually reaching them via oVirt's host.
 
-### Owner
+## Owner
 
 *   Name: Greg Padgett <gpadgett@redhat.com>
 *   Name: Amit Aviram <aaviram@redhat.com>
 *   Name: Nir Soffer <nsoffer@redhat.com>
 
-### Detailed Description
+## Detailed Description
 
-#### Goals
+### Goals
 
-1.  Upload VM Disk Image to Storage Domain
-2.  Upload ISO Image to ISO Domain
-3.  Download VM Disk Image from URL to Storage Domain
-4.  Download ISO Image from URL to ISO Domain
-5.  (soon) Provide a [REST] API to upload and/or perform random I/O on disk images on the storage
+1.  Upload VM images to oVirt Storage.
+3.  Upload VM images from a URL to oVirt Storage.
+4.  Download oVirt Images.
+5.  Provide a [REST] API to upload/download and/or perform random I/O on disk images on the storage.
 6.  (future) Import entire OVA
 7.  (future) Import and convert (v2v) vmdk
 
 Each supported operation will be available in Webadmin and the REST API. Select operations, such as VM disk image upload, may be available in the User Portal in future releases. REST API users will want to use a script or other automation for ease of use.
 
-The implementation will support uploading files much larger than a typical HTTP request, on the order of several (even hundreds) of GB or more, without relying on launching external programs, processes, tools, etc.
-
-#### Limitations
+### Limitations
 
 *   IE 10, Firefox 3.6, or Chrome 13 or greater are required to upload via Webadmin due to lack of necessary HTML5 APIs in older browsers. The UI will detect this and enable upload only if supported.
 
-#### User Experience
+### User Experience
 
-The upload action for a VM disk will be initiated and controlled in Webadmin by navigating to Storage > <file/block domain> > Disks and using the control buttons present; ISO upload will be under Storage > <ISO domain> > Images.
+The upload action for a VM disk is initiated and controlled in Webadmin by navigating to Storage > <file/block domain> > Disks, or Disks tab- And using the control buttons present; ISO upload will be under Storage > <ISO domain> > Images.
```

Comment:
It doesn't render to anything- just shows this text. I will **bold** it though, or mark it somehow.